docs/Documentation/UserGuide/9-Tools-spark-iotdb.md

```diff
@@ -0,0 +1,141 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+## version
+
+The versions required for Spark and Java are as follow:
+
+| Spark Version | Scala Version | Java Version | TsFile |
+| ------------- | ------------- | ------------ |------------ |
+| `2.4.3`        | `2.11`        | `1.8`        | `0.9.0-SNAPSHOT`|
+
+
+## install
+mvn clean scala:compile compile install
+
+
+# 1. maven dependency
+
+```
+    <dependency>
+      <groupId>org.apache.iotdb</groupId>
+      <artifactId>spark-iotdb-connector</artifactId>
+      <version>0.9.0-SNAPSHOT</version>
+    </dependency>
+```
+
```

Comment:
Since iotdb-jdbc is also required, should we add the maven dependency of jdbc here too?
```
    <dependency>
      <groupId>org.apache.iotdb</groupId>
      <artifactId>iotdb-jdbc</artifactId>
      <version>0.9.0-SNAPSHOT</version>
    </dependency>
```

Reply:
As far as I'm concerned, since the connector depends on jdbc. Depending on the connector will automatically depend on jdbc.